# x pointing forward and y pointing left
x_coordinate: [3, 3, 0 ,0, 3, 3, 0]
y_coordinate: [0, -3, -3, -6, -6, -9, -9]
